bokeh/bokeh
# Ability to save as svg

Partially fixes: #538 

To test:
1. Run any file example
2. Add `?svg=1` to the end of the html file
3. The canvases will go blank
4. Click the `preview/save` button - it will download the svg of the plot

To Do:
- [x] display either the original canvas or the svg'd canvas in the whitespace (not sure why this is happening)

Notes:
* canvas2svg apparently works with node-canvas, so if the other work getting node-canvas integrated works, then we should be able to programatically get out png or svg
* the addition of canvas2svg increases bokeh.js from 1649KB to 1691KB (an increase of 42KB)
